build.gradle

```diff
@@ -66,14 +66,15 @@ allprojects {
         compile "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:$jacksonDataTypeVersion"
         compile "com.google.guava:guava:$guavaVersion"
         compile 'commons-net:commons-net:3.6'
-
-        testCompile "junit:junit:$junitVersion"
-        testCompile "org.testfx:testfx-core:$testFxVersion"
-        testCompile "org.testfx:testfx-junit:$testFxVersion"
-        testCompile "org.testfx:testfx-legacy:$testFxVersion", {
+        compile "org.testfx:testfx-core:$testFxVersion"
+        compile "org.testfx:testfx-junit:$testFxVersion"
+        compile "org.testfx:testfx-legacy:$testFxVersion", {
```

Comment:
Can you explain the reasoning behind changing `testCompile` to `compile`? This would cause the dependencies required for the tests to not be compiled during tests, no?

Reply:
i need to use the FxRobot class in testfx library for displaying the selected text in command box
it will still be compiled during tests

Reply:
Why not add a new compile import instead of deleting testCompile? That is what I don't get. You are essentially removing the dependency for the test build?